.ci_support/migrations/cuda120.yaml

```diff
@@ -2,12 +2,8 @@ migrator_ts: 1682985063
 __migrator:
   kind:
     version
-  # Vendor CUDA 12 migrator to use GCC 11.
-  # This is needed to workaround a pybind + GCC 12 + nvcc 12 bug
-  # xref: https://github.com/pybind/pybind11/issues/4606
-  use_local: true
```

Comment:
Think we need to keep this change to workaround the pybind11 + GCC 12 + NVCC 12 bug